index.js

```diff
@@ -532,28 +571,35 @@
       $types[k] = {extractor: K([]), type: t};
     });
 
-    return new _Type(FUNCTION, '', '', format, test, $keys, $types);
+    return new _Type(FUNCTION, '', '', format,
+                     AnyFunction, K(true), $keys, $types);
```

Comment:
If it's not possible to have all the arguments on one line, please start each one at the same column. ;)